packages/closer/pages/events/[slug]/report.tsx

```
@@ -0,0 +1,330 @@
+  }
+
+  if (!canViewReport) {
+    return <PageNotAllowed />;
+  }
+
+  // Calculate earnings after VAT
+  const totalRevenue = eventRevenue + bookingRevenue;
+  const vatAmount = eventRevenue * vatRate;
+  const earningsAfterVAT = eventRevenue - vatAmount;
+
+  const start = event && event.start && dayjs(event.start);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)